src/compiler/crystal/semantic/method_lookup.cr

```diff
@@ -1,5 +1,55 @@
 require "../types"
 
+# Looking up matches involves two steps:
+#
+# 1. Lookup is done without autocasting disabled.
```

Comment:
```suggestion
# 1. Lookup is done with autocasting disabled.
```

Reply:
Thanks, it happened because I changed how it's phrased a couple of times.